recipes-tag/ocl-icd/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "ocl-icd" %}
+{% set version = "2.2.12" %}
+{% set sha256 = "17500e5788304eef5b52dbe784cec197bdae64e05eecf38317840d2d05484272" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/OCL-dev/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+  # Makes ocl-icd install the OpenCL headers it ships
+  - install-headers.patch
+  # Recursively check for icds in vendor dir
+  - check_icds_recursively.diff
+
+build:
+  number: 1003
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - pkg-config
+    - libtool
+    - automake
+    - autoconf
+    - {{ compiler('c') }}
+    - ruby
```

Comment:
This is added by me on top of the [conda-forge's recipe](https://github.com/conda-forge/ocl-icd-feedstock/blob/22625432a0ae85920825dfeb103af9fe7bd6a950/recipe/meta.yaml#L23-L29).